### ISSUE TITLE
WD My Book Essential 1 TB Decryption of my fathe who passed away

### ISSUE THREAD

**Issue by MrNeal01** [themaddoctor/linux-mybook-tools#66]
Hi,

I lost my father in 2016. He left me this wd my book harddrive but it is encrypted. I have also another wd harddrive but i found its password. I couldn't find the other harddrives' password which has contain my all childhood photos with my father and my family and I am so desperatefull about that. 

I can proove that this is my fathers' harddrive. I can send you the certificate of death and Certificate of legal owner. Also, I have the first date of registration of this harddrive. I can give any proove. 

About the Drive:

It is WD10EADS-11M2B1

**Comment by themaddoctor** [comment]
Post your documents or put them on a google drive.
I cannot give you my email address.

**Comment by MrNeal01** [comment]
https://drive.google.com/drive/folders/1apbyzgEN4iAUPxZrvX2Qe2gqk21oAwqn?usp=sharing

**Comment by themaddoctor** [comment]
Sorry, email and password required by that link.
I do not have a google account, and will not make one.
Set your folder to public or find some other way.

**Comment by MrNeal01** [comment]
can you re-click I change the sharing settings

https://drive.google.com/drive/folders/1apbyzgEN4iAUPxZrvX2Qe2gqk21oAwqn?usp=sharing

**Comment by themaddoctor** [comment]
Received.

What is the chip number on the USB-SATA bridge card?

**Comment by MrNeal01** [comment]
because the screws just can not openning at all. I tried so many thing

**Comment by MrNeal01** [comment]
I add an image is that what you ask?

**Comment by themaddoctor** [comment]
For that chip, I would do a dictionary attack.
If you want something faster, you could contact the authors of this paper:
https://eprint.iacr.org/2015/1002.pdf
They have a program that knows how the keys were created at the factory.

For a dictionary attack, I need the keyblock and sector 0 from the drive. See the PDF in this project for instructions.
Also need a list of possible passwords.

**Comment by MrNeal01** [comment]
I'm deeply sorry but I have no basic knowledge about all this stuff. Would you mind if you can write step by step for me. Where should I start? What should I do?

**Comment by themaddoctor** [comment]
No, I wrote the PDF so I could avoid that. Please find someone locally who can help you through it.
